### PR TITLE
Move starcounter-include on the template root

### PR DESCRIPTION
**Pull request by Mihaiii**
For Issue #23 .

### CODE REVIEW

File: src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html [hunk 1]
```diff
@@ -15,12 +15,11 @@
...
         </nav>
-        <div class="form-horizontal cashier-current-page">
-            <starcounter-include partial="{{model.CurrentPage}}" slot="cashier/current-page"></starcounter-include>
-        </div>
     </template>
     <template is="declarative-shadow-dom">
         <slot name="cashier/navigation-bar"></slot>
-        <slot name="cashier/current-page"></slot>
+        <div class="form-horizontal cashier-current-page">
+            <starcounter-include partial="{{model.CurrentPage}}"></starcounter-include>
```
Comment (PatrykSzwer) [inline src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html]: See comment above.

Comment (Mihaiii) [inline src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html]: Just had another look at what I did. :))
My bad, sorry, will fix.

**Comment by Mihaiii** [comment]
You're right :)).
Please have another look.

**Comment by Mihaiii** [comment]
Done.

File: src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html [hunk 2]
```diff
@@ -15,12 +15,15 @@
...
             </ul>
         </nav>
-        <div class="form-horizontal cashier-current-page">
-            <starcounter-include partial="{{model.CurrentPage}}" slot="cashier/current-page"></starcounter-include>
-        </div>
+        <starcounter-include slot="cashier/current-page" partial="{{model.CurrentPage}}"></starcounter-include>
     </template>
     <template is="declarative-shadow-dom">
+        <style>
+            @import url("/Cashier/css/master.css")
```
Comment (PatrykSzwer) [inline src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html]: `cashier-current-page` class is not in the master.css file, It is in standalone.css file.

Comment (Mihaiii) [inline src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html]: No, is in `master.css` on` develop` branch.
I think you're looking on `master` branch.

Comment (PatrykSzwer) [inline src/Cashier/wwwroot/Cashier/viewmodels/MasterPage.html]: Ou, You're right, my bad.

**Pull request merged**